starter-workflows/README.md

```diff
@@ -0,0 +1,12 @@
+## Starter Workflows
+
+The Frogbot Code Scanning workflow is configured in this directory.
+Two files can be edited: frogbot-scan-pr.yml, which configures the scan workflow, and forgbot-fix.yml, which opens a new
+Pull Request with the necessary changes if a vulnerability is found.
+To start using Starter-Workflows please refer the GitHub documentation:
+https://docs.github.com/en/actions/using-workflows/using-starter-workflows
+
+## Frogbot Contributors
+
+Whenever you make any changes to files in this directory, you should update them in this repository as well:
```

Comment:
...in this repository as well:
-->
...in the following repository as well, so that they become public.

Reply:
done